examples/run_examples.sh

```diff
@@ -72,7 +72,8 @@ do
         echo "*** Example $example failed."
         failed_examples="$failed_examples $example"
     fi
-    rm -rf *vtu *sqlite *pkl *-journal restart_data
+    # FIXME: This could delete data from other runs
+    # rm -rf *vtu *sqlite *pkl *-journal restart_data
```

Comment:
This bit me once when I had a bunch of output files from a longer run of one of the examples (they got removed in addition to the ones produced by `run_examples.sh`). Maybe it would be better for the `rm` part of this to be in the CI script somewhere?